tox.ini

```diff
@@ -39,8 +39,8 @@ commands =
 basepython = python3
 allowlist_externals = git
 commands =
-  ruff check qiskit test tools examples setup.py
-  black --check {posargs} qiskit test tools examples setup.py
+  ruff check qiskit test tools examples setup.py qiskit_pkg/setup.py
+  black --check {posargs} qiskit test tools examples setup.py qiskit_pkg/setup.py
```

Comment:
Super minor, but tbh all the places where we've said `qiskit_pkg/setup.py` in `tox.ini`, we could probably just have said `qiskit_pkg` which is a bit more defensive against expansion.

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/10530/commits/db80f2ad51c5da0df6e41c8d2cf9fc15228151d3